### ISSUE TITLE
Merge into origional repo, preserve history

### ISSUE THREAD

**Issue by minermoder27** [ksjdragon/equivalent_exchange#9]
Hello,

I have merged this repo into the main EE repo, and have found that this repo shares no common ancestor commits. This makes it very hard to merge or rebase changes. Is it possible to insert those commits into this repo?

**Comment by ksjdragon** [comment]
Which commits are you talking about?

**Comment by ksjdragon** [comment]
Nvm, it's mainly because so much of the code was rewritten, that i decided to scrap a lot. Then again, might have been my git messing up.. again.

**Comment by minermoder27** [comment]
see https://github.com/minermoder27/equivalent_exchange/commits/master?page=2, the first 3 commits (which I wrote) are not included in your git repo. Most likely, it was Git's fault. It seems like you cloned my repo, git lost all it's history, and then you accidentally recommitted everything that was already there, not just the changes. Using grafts, I was able to fix it, however it means that GitHub does not understand any of it, because it changed the parents of your commits, the hash changed. Unfortunately, these headaches (as you seem to have found) are what comes of using a distributed VCS (Git).

**Comment by ksjdragon** [comment]
Well, i suppose that's fine then. lol. Though, if you're barely active probably not going to work on this, why did you even bother merging in the first place?

**Comment by minermoder27** [comment]
Mainly just the forum thread: https://forum.minetest.net/viewtopic.php?pid=133779
While I could have linked to this repo instead, I prefer having the 'master copies' in my own repos.

**Comment by minermoder27** [comment]
Also, I have added you as a collaborator for my repo. If you don't want to use it, that's fine, tell me and I'll link to this repo in the forum.

**Comment by ksjdragon** [comment]
Right.. but are you going to be active enough taht you'll accept pull requests, if I ever go and try to make some more of it.

**Comment by ksjdragon** [comment]
Ah, nvm, your message sent after i sent that one. you might want to change the images on the forum page though. There were quite a bit of textures added in.

**Comment by ksjdragon** [comment]
And, maybe, a few descriptions, if it matters. Because i ended up changing plenty of values and random stuff. Sorry to make my messages so weirdly sporadic.

**Comment by minermoder27** [comment]
Yeah, just probably not doing much or any dev work. But I'll certainly merge pull requests or help you with something, if you ask me. Just probably not much dev work by my self. I may do some, though.

**Comment by minermoder27** [comment]
Ok
